src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceBatchSource.java

```diff
@@ -125,7 +125,7 @@ public void transform(KeyValue<Schema, Map<String, String>> input,
    */
   @Path("getSchema")
```

Comment:
this annotation can be removed. In 6.0.0, the UI now gets the schema based on whatever is set in configurePipeline().